### DISCUSSION TITLE
Procedure for implementing new CV/method -- developer documentation

### DISCUSSION THREAD

**Discussion by sivadasetty** [SSAGESLabs/PySAGES#179] [General]
May be this is already considered but want to add it here. Wondering if we should list clearly the steps to implement a new CV or method in PySAGES user documentation. It would be nice to have a checklist especially on steps related to a) adding example systems, b) updating pysages tests, c) updating docs, and d) ci workflow.

**Comment by p-zubieta** [comment]
We should definitely add this. I'll make this into an issue so we keep track of it.
